README.md

```diff
@@ -1,2 +1,56 @@
-# bubble-sorting
-This is a Ruby code that allows you to sort an array using the bubble sorting method.
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+## Ruby Bubble-Sort and  Bubble-Sort_by project
+
+> This is a project made with ruby to order items inside of a array, one order items with numbers and the other one order strings inside of the array using a yield feature.
+
+## Built With
+
+- Ruby
+
+## Getting Started
+
+Be sure to have a github account and that this account is connected locally on your PC
+
+### Setup
+
+Create a new folder/directory for the project
+
+### Install
+
+- open your terminal on your OS
+- go to the desired folder/directory that you want to have the project in
+- git clone gh repo clone https://github.com/carlos-osorio-developer/bubble-sorting/tree/bubblesort
```

Comment:
It appears you have made a typo here - `git clone gh repo clone ` -, please fix this typo.